Assets/_BForBoss/_Core/Scripts/Editor/LevelDesignFeedback/LevelDesignFeedbackEditorWindow.cs

```diff
@@ -0,0 +1,213 @@
+using System;
+using System.Text;
+using BForBoss;
+using Trello;
+using UnityEditor;
+using UnityEngine;
+
+public class LevelDesignFeedbackEditorWindow : EditorWindow
+{
+    private const float ELEMENT_SPACING = 5f;
+    private const int NUMBER_OF_FEEDBACK_LINES = 5;
+
+    public Action OnWindowClosed;
+    
+    private static LevelDesignFeedbackEditorWindow _window = null;
+
+    private static Texture2D _originalImage;
+    private static Texture2D _image;
+    private string _title;
+    private string _feedback;
+    
+    public static LevelDesignFeedbackEditorWindow OpenWindow(Texture2D screenshot)
+    {
+        _originalImage = screenshot;
+        _image = CreateTextureCopy(_originalImage);
+        _window = (LevelDesignFeedbackEditorWindow) GetWindow(typeof(LevelDesignFeedbackEditorWindow));
+        _window.titleContent = new GUIContent("Add Level Design Feedback");
+        _window.minSize = new Vector2(250, 500);
+        _window.maxSize = new Vector2(400, 700);
+        _window.Show();
+
+        EditorApplication.isPaused = true;
+
+        return _window;
+    }
+    
+    private static Texture2D CreateTextureCopy(Texture2D sourceTexture)
+    {
+        Texture2D textureCopy = new Texture2D(sourceTexture.width, sourceTexture.height, TextureFormat.ARGB32, false);
+        textureCopy.SetPixels(sourceTexture.GetPixels());
+        textureCopy.Apply();
+
+        return textureCopy;
+    }
+
+    private void OnGUI()
+    {
+        using (new EditorGUILayout.VerticalScope())
+        {
+            DrawPreviewImage();
+
+            EditorGUILayout.Space(ELEMENT_SPACING);
+
+            DrawDescription();
+            
+            GUILayout.FlexibleSpace();
+
+            DrawButtons();
+            
+            EditorGUILayout.Space(ELEMENT_SPACING);
+        }
+    }
+
+    private void DrawPreviewImage()
+    {
+        Rect previewImageRect;
+        EditorGUILayout.Space(ELEMENT_SPACING);
+
+        using (new EditorGUILayout.VerticalScope())
+        {
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                GUILayout.FlexibleSpace();
+                
+                previewImageRect = new Rect(position.width * 0.05f,position.height * 0.02f, position.width * 0.9f, position.height * 0.4f);
+                
+                if (_image == null)
+                {
+                    EditorGUI.DrawRect(previewImageRect, Color.gray);
+                }
+                else
+                {
+                    EditorGUI.DrawTextureTransparent(previewImageRect, _image, ScaleMode.ScaleToFit, 0);
+                }
+
+                if (_image != null && WasElementDoubleClicked(previewImageRect))
+                {
+                    // new SceneViewCameraWindow(SceneView.currentDrawingSceneView)
+
+                    ImageEditorWindow imagePopupWindow = GetWindow<ImageEditorWindow>();
+                    imagePopupWindow.OnWindowClosed += OnImageEdited;
```

Comment:
This seems kinda dangerous, subscribing evrey frame?

Reply:
It only get subscribed if you double click the screenshot

Reply:
Does it get unsubscribed? does it need to be unsubscribed?

Reply:
Im pretty sure it doesnt cause the reference should get destroyed when the window gets destroyed, but to be safe I will unsubscribe it in `OnImageEdited` to be safe, since it is static and could persist

Good catch